pkg/schemas/schemas_test.go

```diff
@@ -0,0 +1,99 @@
+package schemas
+
+import (
+	"encoding/json"
+	"path/filepath"
+	"testing"
+
+	"github.com/xeipuuv/gojsonschema"
+
+	crdpkg "fybrik.io/json-schema-generator/testPkgs/crd"
+	schemapkg "fybrik.io/json-schema-generator/testPkgs/schemaPkg"
+)
+
+func TestValidApp(t *testing.T) {
+	schemaPath := "../../testdata/schema/sample_crd.json"
```

Comment:
I don't see this file

Reply:
This file is created when running `generate-test-data` in the Makefile.

Reply:
no, it doesn't, probably due to errors during schema generation.

Reply:
Are you getting the same errors when you run `make test` after running `make build-tool`?

Reply:
No, after running `make build-tool`, I don't get the errors. But in this case you have to specify the dependency  in the Makefile

Reply:
I added it to `make test`.